src/app/components/job/action/add/job-manager-task-picker.component.ts

```diff
@@ -34,6 +36,7 @@ export class JobManagerTaskPickerComponent extends JobTaskBaseComponent {
         this._baseFormControls["displayName"] = [null];
         this._baseFormControls["killJobOnCompletion"] = [true];
         this._baseFormControls["runExclusive"] = [true];
+        this._baseFormControls["requiredSlots"] = [1, [Validators.required, Validators.min(1)]];
```

Comment:
We may want to add a max value validator based on pool taskSlotsPerNode settings, but it can be done in another PR.

Reply:
The max value validator based on pool taskSlotsPerNode would be based on the VM sizes chosen for that specific pool right? Do you know how that is determined? Like if there is information that I can pull that has a correspondence between the VM size and the max value for the taskSlotsPerNode?

Reply:
The max value for taskSlotsPerNode is 4 x number of VM cores, when vmSize is not specified, the max value is 256